underscore.js

```diff
@@ -433,9 +433,15 @@
   });
 
   // Safely create a real, live array from anything iterable.
+  var reStrSymbol = /[^\ud800-\udfff]|[\ud800-\udbff][\udc00-\udfff]|[\ud800-\udfff]/g;
   _.toArray = function(obj) {
     if (!obj) return [];
     if (_.isArray(obj)) return slice.call(obj);
+    if (_.isString(obj)) {
+      // Keep surrogate pair characters together
+      // https://mathiasbynens.be/notes/javascript-unicode
```

Comment:
I don't think it's necessary. The regexp itself is not taken from there. The "keep surrogate pairs together" comment is sufficient.